scripts/prepareDeploy.sh

```diff
@@ -70,11 +70,11 @@ flow=`getLatest flow $flow`
 ## Modify poms with the versions to release
 echo "Setting version=$version to vaadin-flow-components"
 mvn -B -q versions:set -DnewVersion=$version || exit 1
-setPomVersion flow $flow
-setPomVersion vaadin-flow-components-shared $version || exit 1
```

Comment:
Just to double check (I have no idea how the version constants in the poms are updated), setting the version for the new modules is not required here?